app/openlxp_xds_project/settings.py

```diff
@@ -171,6 +173,50 @@
 
 AUTH_USER_MODEL = 'core.XDSUser'
 
+# openlxp_authentication settings
+SOCIAL_AUTH_STRATEGY = 'openlxp_authentication.models.SAMLDBStrategy'
+JSONFIELD_ENABLED = True
+USER_MODEL = 'core.XDSUser'
+SESSION_EXPIRATION = True
+if(os.environ.get('LOGIN_REDIRECT_URL') != None):
+    LOGIN_REDIRECT_URL = os.environ.get('LOGIN_REDIRECT_URL')
+
+if(os.environ.get('OVERIDE_HOST') != None):
+    OVERIDE_HOST = os.environ.get('OVERIDE_HOST')
+    BAD_HOST = os.environ.get('BAD_HOST')
+
+if(os.environ.get('STRATEGY') != None):
+    SOCIAL_AUTH_STRATEGY = os.environ.get('STRATEGY')
+
+SP_ENTITY_ID = os.environ.get('ENTITY_ID')
+
+SP_PUBLIC_CERT = os.environ.get('SP_PUBLIC_CERT')
+SP_PRIVATE_KEY = os.environ.get('SP_PRIVATE_KEY')
+ORG_INFO = {
+    "en-US": {
+        "name": "example",
+        "displayname": "Example Inc.",
+        "url": "http://localhost",
+    }
+}
+TECHNICAL_CONTACT = {
+    "givenName": "Tech Gal",
+    "emailAddress": "technical@localhost.com"
+}
+SUPPORT_CONTACT = {
+    "givenName": "Support Guy",
+    "emailAddress": "support@localhost.com",
+}
+USER_ATTRIBUTES = ["user_permanent_id",
```

Comment:
Style nit: Unless we have to place the initial attribute on the first line of the declaration it would be better to have it on its own line like the rest of the attributes.